sites/svelte.dev/src/routes/docs/+server.js

```diff
@@ -0,0 +1,13 @@
+// @ts-expect-error custom suffix doesn't have types
+import js from './redirect.js?minified';
+
+// prerenderer will choke otherwise
```

Comment:
ah, ok. I can't think of a solution to that other than to make it `docs/index.html`. or maybe that's what the `docs.html` was for, but in that case could could name this something else I think so that prerendering does work
```suggestion
// avoid outputting a file named "docs" that would conflict with prerendered "docs" directory
```